app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -243,6 +169,129 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
         }
     }
 
+    private fun setScreenSize(isTablet: Boolean) {
+        if (isTablet == this.isTablet) return
+
+        if (isTablet) {
+            addTabletActions(requireContext())
+        } else {
+            removeTabletActions()
+        }
+
+        this.isTablet = isTablet
+    }
+
+    @Suppress("LongMethod")
+    private fun addTabletActions(context: Context) {
+        val enableTint = ThemeManager.resolveAttribute(R.attr.textPrimary, context)
+        val disableTint = ThemeManager.resolveAttribute(R.attr.textDisabled, context)
+        backAction?.let {
```

Comment:
Let's add a new line above this. Since the `backAction` is rather big, it could use the separation for easier readability.